pkg/helpers/azureconst.go

```diff
@@ -464,44 +512,48 @@ func GetKubernetesAllowedVMSKUs() string {
         "Standard_E16-4s_v3",
         "Standard_E16-8s_v3",
         "Standard_E16_v3",
-        "Standard_E16a_v3",
```

Comment:
is that legit? some skus are gone? @mboersma @palma21

Reply:
looks legit.. https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes-memory#eav4-series